packages/firestore/src/model/proto_values.ts

```diff
@@ -0,0 +1,385 @@
+/**
+ * @license
+ * Copyright 202 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '../protos/firestore_proto_api';
```

Comment:
Since we already have an "api" package this is potentially confusing. Consider importing this as `protos` or something that more concretely points toward this being the messages.